src/abstractdataframe/abstractdataframe.jl

```diff
@@ -390,21 +390,23 @@ describe(io, df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :nmissi
 * `io` : optional output descriptor
 * `stats::AbstractVector{Symbol}`: the summary statistics to report. Allowed 
   fields are `:mean`, `:std`, `:min`, `:q25`, `:median`, `:q75`, `:max`, `:eltype`, 
-  `:nunique`, and `:nmissing`
+  `:nunique`, `:first`, `:last`, `:nmissing`, and `:all`
```

Comment:
`:all` is no longer allowed as a field. Also should use `stats::Union{Symbol,AbstractVector{Symbol}}`.